tests/zquantum/qaoa/estimators/cvar_estimator_test.py

```diff
@@ -87,3 +93,26 @@ def test_cvar_estimator_returns_correct_values(self, estimator, backend, operato
         # Then
         assert len(expectation_values) == len(estimation_tasks)
         assert expectation_values[0].values == pytest.approx(target_value)
+
+    def test_cvar_estimator_returns_correct_values_without_sampling(
```

Comment:
Instead of having this test, what about change the `estimator` fixture to:

```
@pytest.fixture(
    params=[
        {
            "alpha": 0.2,
            "use_exact_expectation_values": False,
        },
        {
            "alpha": 0.2,
            "use_exact_expectation_values": True,
        },

    ]
)
def estimator(request):
    return CvarEstimator(**request.param)
```
?
It would require some more test cases, but I think we can limit ourselves to 3 values of alpha.